lib/reline/line_editor.rb

```diff
@@ -1004,7 +1005,7 @@ def wrap_method_call(method_symbol, method_obj, key, with_operator = false)
       method_obj = method(method_symbol)
     end
     if @vi_arg
-      if key.match?(/\A\d\z/)
```

Comment:
With the change in this pull request, inserting number with quoted_insert(`\C-v9`) and bracketed paste(`\e[200~9\e[201~`) has `key="9"` that wrongly match to `/\A\d\z/`.
We need to use method_symbol instead of this regexp.